src/zope/interface/registry.py

```diff
@@ -184,6 +145,9 @@ def __init__(self, name='', bases=()):
         self._init_registrations()
         self.__bases__ = tuple(bases)
 
+        if hasattr(self, '_v_utility_registrations_cache'):
```

Comment:
I really don't think people pickling these things outside the realm of ZODB is likely.  Even if they do, there is no harm, just some extra data.